src/docs/environment.mdx

```diff
@@ -180,6 +185,12 @@ If you would like to be able to run `devserver` outside of your root checkout, y
 
 ---
 
+**Problem:** Some kind of python version issues like `python-build: definition not found ...` or `pyenv: version 'x.x.x' is not installed` 
+
+**Solution** Make sure you have the correct python versions installed. Go to the git root directory and run `make setup-pyenv`.
```

Comment:
Since the dev env docs are now updated, there's a very low chance anyone would run into this issue. So I'd recommending just removing this to keep things slimmer.

Or, you could just change the solution to "Run `make setup-pyenv`." People know make is in git root.

Reply:
Good point, I can remove it, we can always add it back if lot's of people end up hitting the issue.